### ISSUE TITLE
prefer doxygen @copydoc over duplicating documentation

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#944]
We have many duplicated lines of documentation, especially in the material models, that doesn't make a lot of sense. Example: simple.h is_compressible().

**Comment by pmbremner** [comment]
Does the nature of this issue change now that everything is switching over to Sphinx?

**Comment by bangerth** [comment]
No, this is still the case. We are only changing from latex to sphinx for the hand-written manual, but the automatically generated doxygen documentation remains on that system. So what @tjhei points out above remains true.
